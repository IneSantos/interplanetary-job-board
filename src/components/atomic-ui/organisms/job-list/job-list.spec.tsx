import React from "react";
import { render } from '@testing-library/react';
import JobList from "./job-list";

describe("JobList atom tests", () => {
  test(`JobList component render with no list`, () => {
    const { container } = render(
      <JobList jobs={[]}/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
