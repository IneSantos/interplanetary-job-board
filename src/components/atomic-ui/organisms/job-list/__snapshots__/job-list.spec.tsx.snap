// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JobList atom tests JobList component render 1`] = `
<div
  class="jobs-list"
>
  <div
    class="empty-list"
  >
    <h3>
      No results.
    </h3>
    <span>
      Search again or try removing filters.
    </span>
  </div>
</div>
`;

exports[`JobList atom tests JobList component render with no list 1`] = `
<div
  class="jobs-list"
>
  <div
    class="empty-list"
  >
    <h3>
      No results.
    </h3>
    <span>
      Search again or try removing filters.
    </span>
  </div>
</div>
`;
